src/eulalie/lambda.cljc

```diff
@@ -61,6 +63,17 @@
       :headers (body->headers body))
     {:eulalie.lambda/invocation-type invocation-type}))
 
+(defmethod prepare-request :create-function
+  [{:keys [body] :as req}]
+  (assoc req
```

Comment:
This might be easier to read as `(update req :body #(csk-extras/transform-keys csk/->PascalCaseString %))`?